include/boost/geometry/util/math.hpp

```diff
@@ -771,6 +771,125 @@ inline Result rounding_cast(T const& v)
     return detail::rounding_cast<Result, T>::apply(v);
 }
 
+/*!
+\brief Evaluate the sine and cosine function with the argument in degrees
```

Comment:
why is it needed to compute the trigonometric functions with the argument in degrees and not in radians. The input of direct formulas are in radians

Reply:
Actually, the input to Karney direct is in degrees. This can be seen in [direct.cpp](https://github.com/BoostGSoC18/geometry/blob/feature/geodesic_direct/test/formulas/direct.cpp#L44) test file.

Reply:
Currently, the internal functions assume input in degrees. It might be possible to make the switch to radians. However, wouldn't it be simpler if the user could provide input in radians and we convert it into degrees at the beginning? Or, would this be inefficient?

Neither the paper nor GeographicLib documentation go into detail on why this design was chosen. I don't think the computations are any faster because of this, because the angles are manually converted into radians at various points in the code.

Reply:
>Currently, the internal functions assume input in degrees. It might be possible to make the switch to radians. However, wouldn't it be simpler if the user could provide input in radians and we convert it into degrees at the beginning? Or, would this be inefficient?

I am not sure that I understand what you want to say. The user can use either radians or degrees and this is converted to radians and passed to formulas for computations.

>Neither the paper nor GeographicLib documentation go into detail on why this design was chosen. I don't think the computations are any faster because of this, because the angles are manually converted into radians at various points in the code.

If there are a lot of conversions it should be investigated whether this can be avoided by replacing the current function by one that uses only radians. Maybe in a separate PR.

Reply:
@vissarion So, I will perform the conversion to radians in a separate PR. I have created an [issue](https://github.com/BoostGSoC18/geometry/issues/2) as a reminder for myself.

If there are no further changes required with this PR, can we please merge this?